adage.snakefile

```diff
@@ -0,0 +1,36 @@
+# This snakefile relies on the outputs from add_new_to_compendia.snakefile,
+# specifically the file outputs/filt_norm_compendia/*compendium_p2_filtered_num_reads.csv
+# It uses this file as well as sets of samples defined in the R script to perform ADAGE analysis.
+
+STRAIN = ['pao1', 'pa14'] # only run pao1 since this is what the eADAGE model is based on
+CONTROLS = ['asm', 'm63', 'lb']
+GROUPS = ['spu', 'spu_m', 'spu_pub']
+
+rule all:
+    input:
+        expand("outputs/adage/{strain}-{group}-{control}_combined_activation_and_assoc_df.tsv", strain = STRAIN, control = CONTROLS, group = GROUPS)
+
+rule install_adage:
+    output: install = "outputs/adage/adage_installed.txt"
+    conda: "envs/adagepath.yml"
+    script: "scripts/snakemake_install_ADAGEpath.R"
+
+rule run_adage:
+    input:
+        install = "outputs/adage/adage_installed.txt",
+        num_reads = "outputs/filt_norm_compendia/{strain}_aligned_compendium_p2_filtered_num_reads.csv"
+    output: 
+        data_activity = "outputs/adage/{strain}_{group}_{control}_data_activity.tsv",
```

Comment:
What's the point of listing all of these outputs if the all rule is only looking for combined_activation_and_assoc_df.tsv? Is it for the benefit of the human reader?

Reply:
Honestly it's just me being lazy :joy: I need all of these outputs so I want the rule to fail if one of them isn't written, but I didn't want to bother typing them all out for the rule all. But I think they're all potentially important for biological interpretation so I made them all.

I need them listed in the rule bc I use the names to subset the `snakemake@output` list in R to determine which file is output where.